src/dyndns.py

```diff
@@ -154,14 +179,18 @@ def dyndns_subscribe(operation_logger, domain=None, key=None):
             error = f'Server error, code: {r.status_code}. (Message: "{r.text}")'
         raise YunohostError("dyndns_registration_failed", error=error)
 
+    # Set the domain's config to autopush
+    from yunohost.domain import domain_config_set
+    domain_config_set(domain, key="dns.zone.autopush", value=1)
```

Comment:
Something needs to be designed for this to be applied to all dyndns domains on existing instances somehow

Reply:
all dyndns with a cron on the system